# HG changeset patch
# Parent  56fc7e0ad22e9624c8210ecb6aa5c0443ff06cfc
Stop documenting the -f option

We might want to deprecate it at some point.

diff --git a/core/lightsd.c b/core/lightsd.c
--- a/core/lightsd.c
+++ b/core/lightsd.c
@@ -155,7 +155,6 @@
 "                                       repeated).\n"
 "  [-s,--socket /unix/socket]           Open an Unix socket at this location\n"
 "                                       (can be repeated).\n"
-"  [-f,--foreground]                    Stay in the foreground (default).\n"
 "  [-d,--daemonize]                     Fork in the background.\n"
 "  [-u,--user user]                     Drop privileges to this user (and the\n"
 "                                       group of this user if -g is missing).\n"
diff --git a/dist/lightsd.service b/dist/lightsd.service
--- a/dist/lightsd.service
+++ b/dist/lightsd.service
@@ -3,7 +3,7 @@
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/lightsd -t -v warning -f -u lightsd -s %t/lightsd/socket -c %t/lightsd/pipe
+ExecStart=/usr/bin/lightsd -t -v warning -u lightsd -s %t/lightsd/socket -c %t/lightsd/pipe
 Restart=on-failure
 
 [Install]
diff --git a/docs/first-steps.rst b/docs/first-steps.rst
--- a/docs/first-steps.rst
+++ b/docs/first-steps.rst
@@ -127,7 +127,6 @@
                                           repeated).
      [-s,--socket /unix/socket [+]]       Open an Unix socket at this location
                                           (can be repeated).
-     [-f,--foreground]                    Stay in the foreground (default).
      [-d,--daemonize]                     Fork in the background.
      [-u,--user user]                     Drop privileges to this user (and the
                                           group of this user if -g is missing).
